src/main/java/com/univocity/articles/csvcomparison/parser/Parsers.java

```diff
@@ -37,7 +37,8 @@
 			new OsterMillerParser(),
 			new JacksonParser(),
 			new SimpleFlatMapperParser(),
-			new ProductCollectionsParser()
+			new ProductCollectionsParser(),
+			new MneriCsvParser()
```

Comment:
You should add it to Parsers in the `parsers8` package as your parser will not work with Java 6 or 7 as far as I know (also move the `MneriCsvParser` into that package).